tests/common.go

```diff
@@ -2133,30 +2160,39 @@ func SetSourceKubeConfig() error {
 	if err != nil {
 		return err
 	}
-	SetClusterContext(sourceClusterConfigPath)
-	return nil
+	err = SetClusterContext(sourceClusterConfigPath)
+	return err
 }
 
 // SetDestinationKubeConfig sets current context to the kubeconfig passed as destination to the torpedo test
-func SetDestinationKubeConfig() {
+func SetDestinationKubeConfig() error {
 	destClusterConfigPath, err := GetDestinationClusterConfigPath()
-	expect(err).NotTo(haveOccurred())
-	SetClusterContext(destClusterConfigPath)
+	if err != nil {
+		return err
+	}
+	err = SetClusterContext(destClusterConfigPath)
+	return err
```

Comment:
To be consistent, how about

```
		if err != nil {
			return err
		}
		return nil
```

Reply:
There's no point in doing that, as this error is the last error, which means, there **is no concept** of "early return due to error". This is in fact idiomatic code, as an if statement is not a useful check here, as if it is `nil`, then it will be `nil` and we don't need to modify it.

You suggestion makes sense if this were a function which is constantly modified by people, so it would make sense to have "consistency". But this function might not change ever again, so it's fine.

Reply:
Why don't you just do `return SetClusterContext(destClusterConfigPath)`

Reply:
✅